### PR TITLE
feat: add variable to enable mailjet_rest library

### PR DESCRIPTION
**Pull request by jf-marquis-Adeo**
terraform change to add mailjet option

### CODE REVIEW

**Comment by ttdt** [comment]
I'm unable to get information about failing check (concourse-ci/lint-tests), `details` link is unreachable.

A terraform variable has been added (`mailjet_enabled`) to install `mailjet_rest` library.
I don't know how to manage sendgrid/mailjet specific parameters (auth_key, auth_secret, custom parameters). Do you think it is necessary to merge the PR ? If so, could you provide an example ?

File: main.tf [hunk 1]
```diff
@@ -140,12 +140,13 @@ module "server" {
   server_private           = var.server_private
   cloudsql_proxy_arch      = var.cloudsql_proxy_arch
   cloud_profiler_enabled   = var.cloud_profiler_enabled
+  mailjet_enabled          = var.mailjet_enabled
```
Comment (blueandgold) [inline main.tf]: Can we also update the `README.md` file?

Like the example here?
https://github.com/forseti-security/terraform-google-forseti/pull/297/files

Comment (kevensen) [reply main.tf]: @jf-marquis-Adeo, just FYI, the readme is updated by running `make generate_docs` in the root of the module.

**Comment by kevensen** [comment]
@jf-marquis-Adeo Can you please run `make generate_docs` on your local repo to update the README documentation?

**Comment by ttdt** [comment]
> @jf-marquis-Adeo Can you please run `make generate_docs` on your local repo to update the README documentation?

done

**Pull request merged**